copylot/hardware/lights/abstract_light.py

```diff
@@ -0,0 +1,4 @@
+import abc
+
+class AbstractLight(metaclass=abc.ABCMeta):
```

Comment:
I don't have anything against the name `Light` but are expecting any light source other than lasers? if so we should keep it but if that's not the case, I would propose having the subpackage named `lasers` and this abstract class `AbstractLaser`